deploy/dev/main.tf

```diff
@@ -183,6 +183,37 @@ resource "aws_ecs_task_definition" "insights-connector-gerrit-task" {
 
 }
 
+/* ECS bugzilla connector task definition */
+resource "aws_ecs_task_definition" "insights-connector-bugzilla-task" {
```

Comment:
you need to add another service for this, like on line no: 271 in your branch in this file.